enterprise-suite/es-monitor-api/bootstrap-monitors.json

```diff
@@ -1,639 +1,738 @@
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "kube node pressure",
+          "description": "{{$labels.node_name}} is in kubernetes condition {{$labels.condition}}"
+        }
+      },
+      "kube_container_restarts": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_pod_container_restarts_rate",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "container restarting rapidly",
+          "description": "container {{$labels.container}} in pod {{$labels.pod}} of {{$labels.es_workload}} restarting rapidly"
+        }
+      },
+      "kube_pod_not_ready": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_pod_ready",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "critical": {
+              "comparator": "<",
+              "threshold": "1"
+            }
+          },
+          "summary": "pod not ready",
+          "description": "pod {{$labels.pod}} on {{$labels.es_workload}} not ready"
+        }
+      },
+      "kube_pod_not_running": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_pod_not_running",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "pod not running",
+          "description": "pod {{$labels.pod}} on {{$labels.es_workload}} not running"
+        }
+      },
+      "kube_workload_generation_lag": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_workload_generation_lag",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "workload not updating",
+          "description": "workload {{$labels.es_workload}} has not updated for 10 minutes"
+        }
+      },
+      "kafka_consumergroup_lag": {
+        "monitorVersion": "1",
+        "model": "growth",
+        "parameters": {
+          "metric": "kafka_consumergroup_lag",
+          "period": "5m",
+          "minslope": "1",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "window": "5m"
+            }
+          },
+          "summary": "consumergroup falling behind",
+          "description": "{{$labels.consumergroup}} is falling behind"
```

Comment:
QQ why don't have we have es_workload for all descriptions ?

Reply:
no descriptions or summaries were changed in this PR, this is just a JSON format change

Here the workload would be "consumer_group_exporter"